reo/src/reopt.jl

```diff
@@ -1052,7 +1051,12 @@ function reopt_run(m, p::Parameter)
     #############  		Outputs    									 #############
     ##############################################################################
 	try
-		results["lcc"] = round(JuMP.objective_value(m)+ 0.0001*value(m[:MinChargeAdder]))
+		if (p.AddSOCIncentive == 1)
```

Comment:
@zolanaj I don't think that we need this if/else block: can we use 
``` m[:REcosts] + 0.0001 * value(m[:MinChargeAdder]```
in both cases?

Reply:
I thought we wanted to remove the incentive when calculating LCC, since it isn't a "real" cost but an artifact of the optimization to obtain reasonable dispatch outcomes like removing then adding the <0.0001 * value(m[:MinChargeAdder]) > back in.  I know the correction wasn't there before, but my impression was that it should be.  Am I missing something?

Reply:
Yes the `REcosts` is the LCC without the SOC incentive in all cases.

Reply:
Ok, then in that case we need to keep the if/else block in; we only add the SOC incentive if the feature is turned on via `p.AddSOCIncentive == 1`, so we should only remove the quantity from the LCC when `p.AddSOCIncentive == 1` as well.

Reply:
Thanks again for fixing my mental block on this @NLaws!  Commit db25d210c138ddf913afbf1cc1c64d40720e07e8 implements your suggested changes.